libs/community/langchain_community/chat_models/tongyi.py

```diff
@@ -158,7 +158,7 @@ def lc_secrets(self) -> Dict[str, str]:
     top_p: float = 0.8
     """Total probability mass of tokens to consider at each step."""
 
-    dashscope_api_key: Optional[SecretStr] = None
+    dashscope_api_key: Optional[SecretStr] = Field(None, alias="api_key")
```

Comment:
integration test fails, believe you need to add a Config class with allow_population_by_field_name=True like this https://github.com/langchain-ai/langchain/blob/cd7abc495a091c41674a284eeccf329651aeec6e/libs/partners/openai/langchain_openai/chat_models/base.py#L327